src/backends/beamcoder.ts

```diff
@@ -246,15 +246,22 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
      * Get imageData for a given time in seconds
      * @param targetTime
      */
-    async getImageDataAtTime(targetTime: number): Promise<ImageData> {
+    async getImageDataAtTime(targetTime: number, target?: Uint8ClampedArray): Promise<ImageData> {
```

Comment:
I thought we had refactored ImageData/canvas out of framefusion 🤔

Reply:
@antoineMoPa , yes, we did. Image data here is just an interfaces defined inside this package